packages/components/header/header.js

```diff
@@ -211,8 +211,14 @@ class Header {
       )
     }
 
-    if (document.body.offsetWidth !== this.width && this.menuIsOpen) {
-      this.closeMobileMenu()
+    // Check for width changes
+    if (this.width !== document.body.offsetWidth) {
+      this.width = document.body.offsetWidth
+
+      // Automatically close menu
+      if (this.menuIsOpen) {
```

Comment:
Not related to your change but I'm wondering whether the auto-closing behaviour actually makes sense here? If you have the menu open and you resize the window, wouldn't you expect it to stay open, but update its contents?